tools/check/go.mod

```diff
@@ -20,3 +20,5 @@ require (
 	gopkg.in/yaml.v2 v2.2.2 // indirect
 	honnef.co/go/tools v0.0.0-20180920025451-e3ad64cb4ed3
 )
+
+go 1.13
```

Comment:
Why we need `go 1.13`?

Reply:
I think it's generated by the go compiler.